base/Enums.jl

```diff
@@ -130,9 +143,9 @@ macro enum(T,syms...)
             end
         end
     end
-    if isa(T,Symbol)
```

Comment:
You're right; I think that's a vestige from when enums could have type parameters.